CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # App Center SDK for Android Change Log
 
-## Version 3.2.3 (Under development)
+## Version 3.2.3
```

Comment:
Please move `### App Center Distribute Play` to this version and update version number to 3.3.0